src/datagen/generated/minecolonies/data/minecolonies/advancements/recipes/food/baked_manchet_bread.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parent": "minecraft:recipes/root",
+  "criteria": {
+    "has_dough": {
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "minecolonies:manchet_dough"
+            ]
+          }
+        ]
+      },
+      "trigger": "minecraft:inventory_changed"
```

Comment:
could we avoid the inventory change triggers? Those are rather laggy and we have like 200+

Reply:
Good question, we should look into them and maybe remove them all if so.